test/judges/test-github-events.rb

```diff
@@ -2332,6 +2332,123 @@ def test_release_event_with_nil_compare_response
     assert_equal([526_301], f.first[:contributors])
   end
 
+  def test_ignores_events_from_configured_bot_users
+    WebMock.disable_net_connect!
+    rate_limit_up
+    stub_github(
+      'https://api.github.com/repos/foo/foo',
+      body: { id: 42, name: 'foo', full_name: 'foo/foo', default_branch: 'master' }
+    )
+    stub_github(
+      'https://api.github.com/repositories/42',
+      body: { id: 42, name: 'foo', full_name: 'foo/foo', default_branch: 'master' }
+    )
+    stub_github(
+      'https://api.github.com/repositories/42/events?per_page=100',
+      body: [
+        {
+          id: '11001',
+          type: 'PushEvent',
+          actor: { id: 1001, login: '0pdd' },
+          repo: { id: 42, name: 'foo/foo' },
+          payload: {
+            push_id: 123,
+            ref: 'refs/heads/master',
+            head: 'abc123'
+          },
+          created_at: '2025-01-01 12:00:00 UTC'
+        },
+        {
+          id: '11002',
+          type: 'IssuesEvent',
+          actor: { id: 1002, login: 'rultor' },
+          repo: { id: 42, name: 'foo/foo' },
+          payload: {
+            action: 'opened',
+            issue: { number: 1 }
+          },
+          created_at: '2025-01-01 12:01:00 UTC'
+        },
+        {
+          id: '11003',
+          type: 'PushEvent',
+          actor: { id: 1003, login: 'human-user' },
+          repo: { id: 42, name: 'foo/foo' },
+          payload: {
+            push_id: 124,
+            ref: 'refs/heads/master',
+            head: 'def456'
+          },
+          created_at: '2025-01-01 12:02:00 UTC'
+        }
+      ]
+    )
+    stub_github('https://api.github.com/user/1001', body: { id: 1001, login: '0pdd' })
+    stub_github('https://api.github.com/user/1002', body: { id: 1002, login: 'rultor' })
+    stub_github('https://api.github.com/user/1003', body: { id: 1003, login: 'human-user' })
+    stub_github(
+      'https://api.github.com/repos/foo/foo/commits/abc123/pulls?per_page=100',
+      body: []
+    )
+    stub_github(
+      'https://api.github.com/repos/foo/foo/commits/def456/pulls?per_page=100',
+      body: []
+    )
+    fb = Factbase.new
+    options = Judges::Options.new({ 'repositories' => 'foo/foo', 'bots' => '0pdd,rultor' })
```

Comment:
@danilax86 `options` var is redundant. Use it directly in `load_it`. For more see [this article](https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html)